sim_module_test_sketch/sim_module_test_sketch.ino

```diff
@@ -0,0 +1,38 @@
+#include <SoftwareSerial.h>
+
+//Create software serial object to communicate with SIM800L
```

Comment:
Will this work with any other sim module models? Or just with SIM800L?

Reply:
I didn`t check, but it should work for all.